server/adapter/src/main/scala/com/eigengo/lift/adapter/AdapterMicroservice.scala

```diff
@@ -24,7 +25,7 @@ object AdapterBoot {
 
 object AdapterMicroservice extends MicroserviceApp(MicroserviceProps("Adapter")) {
 
-  override def boot(implicit system: ActorSystem, cluster: Cluster): BootedNode = {
+  override def boot(config: Config)(implicit system: ActorSystem, cluster: Cluster): BootedNode = {
```

Comment:
Passing `config` to a microservice's `boot()` method as a parameter feels more general. In our case where the config is shared between akka (or the actorsystem) and the Kafka producer I agree it does not matter. It however feels a bit hidden and I thought passing the `config` explicitly for the microservice to have it visibly available without relying on the system would be nicer.

If you think using `system.settings.config` would be a better idea I can change that.